graylog2-web-interface/src/routing/AppFacade.jsx

```diff
@@ -12,7 +12,7 @@ const CurrentUserStore = StoreProvider.getStore('CurrentUser');
 
 import 'bootstrap/dist/css/bootstrap.css';
 import 'font-awesome/css/font-awesome.css';
-import 'stylesheets/newfonts.less';
+import 'opensans-npm-webfont';
```

Comment:
The lookup mechanism in this case works like this: find out it is a module, check the module's directory for `package.json|index.js`, if the first one is found (like in this case) check for a `main` entry and use the file it references. Which is [`style.css` in this case](https://github.com/vaeum/opensans-npm-webfont/blob/master/package.json#L5).